### PR TITLE
Stylelint: Extend order config with new blocks and properties

### PR DESCRIPTION
**Pull request by literat**
Based on https://github.com/visionappscz/stylelint-config/pull/60 and 

The `order` config enforces a consistent order of content in your declaration blocks.:

- Sass variables,
- CSS custom properties,
- Sass @extend,
- single-line Sass @include,
- declarations,
- nested rules,
- any other at-rules.

Based on https://github.com/visionappscz/stylelint-config/pull/61

Add more properties to check their order

Properties added (sorted alphabetically):

* `accent-color`
* `backdrop-filter`
* `backface-visibility`
* `background-blend-mode`
* `border-spacing`
* `box-decoration-break`
* `caret-color`
* `color-scheme`, `color-interpolation`
* `column-fill`, `column-span`
* `contain` (including all long-hand variants)
* `container` (including all long-hand variants)
* `counter-set`
* `direction`
* `empty-cells`
* `font-size-adjust`, `font-variant-*`, `font-feature-settings`, `font-stretch`, `font-synthesis` (including all long-hand variants), `font-kerning`, `font-optical-sizing`, `font-palette`
* `image-orientation`, `image-rendering`
* `inset` (including all long-hand variants),
* `justify-items`
* `list-style-image`
* `mask` (including all long-hand variants), `mask-border` (including all long-hand variants), `mask-type`
* `mix-blend-mode`
* `outline-*`
* `overflow-anchor`
* `overscroll-behavior` (including all long-hand variants).
* `paint-order`,
* `perspective`, `perspective-origin`
* `place-content`, `place-items`, `place-self`
* `quotes`
* `scrollbar-*`
* `table-layout`
* `text-align-last`, `text-indent`, `text-decoration-skip-ink`, `text-underline-*`, `text-rendering`, `text-wrap` (including all long-hand variants)
* `touch-action`,
* `transform-box`, `transform-style`, individual transforms: `translate`, `rotate`, `scale`,
* `transition-behavior`
* `white-space-collapse`
* `word-spacing`

The key is roughly like this: all properties from MDN that are standard, widely supported, non-experimental, not specific for non-western languages, and not designed for paged media.
 
https://developer.mozilla.org/en-US/docs/Web/CSS

### CODE REVIEW

**Comment by literat** [comment]
- [ ] Update package readme accordingly

File: packages/stylelint-config/rules/order.js [hunk 1]
```diff
@@ -1,6 +1,25 @@
...
+        hasBlock: false,
+        name: 'include',
+        type: 'at-rule',
+      },
+      'declarations',
+      'rules',
+      {
+        hasBlock: true,
+        type: 'at-rule',
+      },
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/stylelint-config/rules/order.js]: Ok, will leave this also OFF with the reference to the issue https://github.com/lmc-eu/code-quality-tools/issues/190.

File: packages/stylelint-config/README.md [hunk 2]
```diff
@@ -60,11 +60,90 @@ export default {
...
 
----
+## Checking Properties Order
+
+To further extend control over coding style of your stylesheets, we are also
+checking for properties order.
+
+The `order` config enforces a consistent order of content in your declaration blocks:
+
+1. SASS variables,
```
Comment (adamkudrna) [inline packages/stylelint-config/README.md:77]: Actually it's "Sass" (and "SCSS")

https://sass-lang.com/

**Review by adamkudrna — APPROVED** [review]
I tested the config in Spirit and it works as expected 👍🏻.

File: packages/stylelint-config/README.md (hunk 2 above)
Comment (literat) [reply packages/stylelint-config/README.md]: Ok, I have changed this also in our Textlint config package, so only the `Sass` term is allowed.

File: packages/stylelint-config/README.md [hunk 3]
```diff
@@ -60,11 +60,90 @@ export default {
...
+{
+  "plugins": ["stylelint-prettier"],
+  "rules": {
+    "prettier/prettier": true
+  }
+}
+```
+
+Or, if you feel brave enough and don’t need granular configuration of the stylistic rules,
+you can use [Prettier].
```
[the remark this answers is not in the document]
Comment (literat) [reply packages/stylelint-config/README.md]: Deleted.

**Pull request merged**